frame/oracle/src/lib.rs

```diff
@@ -591,13 +613,17 @@ pub mod pallet {
 					let controller = SignerToController::<T>::get(&answer.who)
 						.unwrap_or_else(|| answer.who.clone());
 
-					let _ = T::Currency::deposit_into_existing(&controller, reward_amount);
+					let result = T::Currency::deposit_into_existing(&controller, reward_amount);
+					if result.is_err() {
+						log::warn!("Failed to deposit {:?}", controller);
```

Comment:
Do we still need this?

Reply:
eh probably not, but it was an item in the audit report, and I can see how this would be a good practice

Reply:
👍